CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+# 0.54.0
+
+Infrastructure:
+* Kotlinx Serialization Json 1.7.1
+* Spring Framework 6.1.12
+* junit-bom 5.11.0
+* SQLite driver 3.46.1.0
+* Kotlinx Datetime JVM 0.6.1
+* Spring Boot 3.3.3
+* PostgreSQL driver 42.7.4
+
+Breaking changes:
+* feat!: EXPOSED-476 Update Kotlin to 2.0.0 by @bog-walk in https://github.com/JetBrains/Exposed/pull/2188
+* refactor!: Move `statementsRequiredForDatabaseMigration` function from `SchemaUtils` to `MigrationUtils` by @joc-a in https://github.com/JetBrains/Exposed/pull/2195
```

Comment:
If this is a breaking change, please also add a point to Breaking-Changes.md. Users may read that document without necessarily looking at the changelog first.